test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -206,6 +206,16 @@ int static inline streaming_node(perf_metrics_t my_info)
     }
 }
 
+static int inline is_streaming_node(perf_metrics_t my_info, int node)
+{
+    if(my_info.cstyle == COMM_PAIRWISE) {
+        return (node < my_info.szinitiator);
+    } else {
+        assert(my_info.cstyle == COMM_INCAST);
+        return true;
```

Comment:
Is that right?  All PEs are streaming in `COMM_INCAST`?

Reply:
Yes, thats what I got from the function above.